packages/cli/test/me.endpoints.test.ts

```diff
@@ -0,0 +1,531 @@
+
+			for (const invalidPayload of INVALID_PATCH_ME_PAYLOADS) {
+				const response = await memberAgent.patch('/me').send(invalidPayload);
+				expect(response.statusCode).toBe(400);
+			}
+		});
+
+		test('PATCH /me/password should succeed with valid inputs', async () => {
+			const member = await Db.collections.User!.findOneOrFail();
+			const memberAgent = await utils.createAgent(app, member);
+
+			const validPayloads = Array.from({ length: 3 }, () => ({
+				password: utils.randomValidPassword(),
+			}));
+
+			for (const validPayload of validPayloads) {
+				const response = await memberAgent.patch('/me/password').send(validPayload);
+				expect(response.statusCode).toBe(200);
+				expect(response.body).toEqual(SUCCESS_RESPONSE_BODY);
+			}
+		});
+
+		test('PATCH /me/password should fail with invalid inputs', async () => {
+			const member = await Db.collections.User!.findOneOrFail();
+			const memberAgent = await utils.createAgent(app, member);
+
+			const invalidPayloads: Array<{ password?: string }> = Array.from({ length: 3 }, () => ({
+				password: utils.randomInvalidPassword(),
+			}));
+
+			invalidPayloads.push({});
+
+			for (const invalidPayload of invalidPayloads) {
+				const response = await memberAgent.patch('/me/password').send(invalidPayload);
+				expect(response.statusCode).toBe(400);
+			}
+		});
+
+		test('POST /me/survey should succeed with valid inputs', async () => {
+			const member = await Db.collections.User!.findOneOrFail();
+			const memberAgent = await utils.createAgent(app, member);
+
+			const validPayloads = [SURVEY, {}];
+
+			for (const validPayload of validPayloads) {
+				const response = await memberAgent.post('/me/survey').send(validPayload);
+				expect(response.statusCode).toBe(200);
+				expect(response.body).toEqual(SUCCESS_RESPONSE_BODY);
+			}
+		});
+	});
+});
+
+const TEST_USER = {
+	email: utils.randomEmail(),
+	firstName: utils.randomName(),
+	lastName: utils.randomName(),
+};
+
+const SURVEY = [
+	'codingSkill',
+	'companyIndustry',
+	'companySize',
+	'otherCompanyIndustry',
+	'otherWorkArea',
+	'workArea',
+].reduce<Record<string, string>>((acc, cur) => {
+	return (acc[cur] = utils.randomString(1, 10)), acc;
+}, {});
+
+const VALID_PATCH_ME_PAYLOADS = [
+	{
+		email: utils.randomEmail(),
+		firstName: utils.randomName(),
+		lastName: utils.randomName(),
+		password: utils.randomValidPassword(),
+	},
+	{
+		email: utils.randomEmail(),
+		firstName: utils.randomName(),
+		lastName: utils.randomName(),
+		password: utils.randomValidPassword(),
+	},
+];
+
+const INVALID_PATCH_ME_PAYLOADS = [
+	{
+		email: 'invalid',
+		firstName: utils.randomName(),
+		lastName: utils.randomName(),
+	},
+	{
+		email: utils.randomEmail(),
+		firstName: '',
+		lastName: utils.randomName(),
+	},
+	{
+		email: utils.randomEmail(),
+		firstName: utils.randomName(),
+		lastName: '',
+	},
+	{
+		email: utils.randomEmail(),
+		firstName: 123,
+		lastName: utils.randomName(),
+	},
+];
```

Comment:
let's think about these:

what about omitting a key?

what about trying to inject `<script>` a http-link or sql-query via a field?